bothub/api/v2/repository/views.py

```diff
@@ -987,3 +987,31 @@ def destroy(self, request, *args, **kwargs):
     def update(self, request, *args, **kwargs):
         self.filter_class = None
         return super().update(request, *args, **kwargs)
+
+    @action(
```

Comment:
to return this data do you need to create a new endpoint? take a look at the Serializer MethodField, I think it does what you want without creating a new route